File: t.java

class t {
	public static int fibonacci(int i) {
		if (i < 2)
			return i;
		return fibonacci(i - 2) + fibonacci(i - 1);
	}
	public static void main(String[] arg) {
		System.out.println(fibonacci(45));
	}
}

File: t.c
#include <stdio.h>

int fibonacci(int i)
{
	if (i < 2)
		return i;
	return fibonacci(i - 2) + fibonacci(i - 1);
}

int main(int argc, char *argv[])
{
	printf("%d\n", fibonacci(45));
}
